src/llama_engine.h

```diff
@@ -1,14 +1,15 @@
 #pragma once
+#include <trantor/utils/AsyncFileLogger.h>
 #include "chat_completion_request.h"
 #include "cortex-common/enginei.h"
+#include "file_logger.h"
+#include "llama.h"
 #include "llama_server_context.h"
 #include "trantor/utils/ConcurrentTaskQueue.h"
-#include "llama.h"
-#include <trantor/utils/AsyncFileLogger.h>
 
 constexpr char log_base_name[] = "logs/cortex";
 constexpr char log_folder[] = "logs";
-constexpr size_t max_log_file_size = 20000000; // ~20mb
+constexpr size_t max_log_file_size = 20000000;  // ~20mb
```

Comment:
Question: is this settable via `.cortexrc`? (and is it per-engine specific?)

Reply:
New logger don't need that config, I'll remove max_log_file_size